board/drivers/can_common.h

```diff
@@ -130,18 +130,24 @@ void can_clear(can_ring *q) {
 
 // Helpers
 // Panda:       Bus 0=CAN1   Bus 1=CAN2   Bus 2=CAN3
-bus_config_t bus_config[BUS_CONFIG_ARRAY_SIZE] = {
```

Comment:
If discarding the 0xFF bus is correct, then `BUS_CONFIG_ARRAY_SIZE` may be removed in favor of `PANDA_CAN_CNT`.

Also, the comment block above is duplicated verbatim in can_common_declarations.h. Probably makes sense to remove the copy here due to otherwise misleading references to CAN MCU interface numbers.